src/Models/Product.php

```diff
@@ -59,6 +59,14 @@ public function getIsActiveAttribute()
         return $this->isActive();
     }
 
+    /**
+     * @return bool
+     */
+    public function isOnStock()
```

Comment:
please set the return type to `bool` and remove the annotation.

Reply:
Done, but use of return annotations and return types are inconsistent through the repository. Maybe we should open an issue or PR about that.

Reply:
True. The reason is that until recently PHP 7.0 was supported which didn't allow nullable return types. Since we've moved to 7.1 with v0.5, return types are the way to go, unless it's not possible.